To: vim_dev@googlegroups.com
Subject: Patch 7.3.254
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.254
Problem:    The coladd field is not reset when setting the line number for a
	    ":call" command.
Solution:   Reset it.
Files:	    src/eval.c


*** ../vim-7.3.253/src/eval.c	2011-07-07 17:36:52.000000000 +0200
--- src/eval.c	2011-07-07 17:51:42.000000000 +0200
***************
*** 3430,3435 ****
--- 3430,3438 ----
  	{
  	    curwin->w_cursor.lnum = lnum;
  	    curwin->w_cursor.col = 0;
+ #ifdef FEAT_VIRTUALEDIT
+ 	    curwin->w_cursor.coladd = 0;
+ #endif
  	}
  	arg = startarg;
  	if (get_func_tv(name, (int)STRLEN(name), &rettv, &arg,
*** ../vim-7.3.253/src/version.c	2011-07-15 21:16:54.000000000 +0200
--- src/version.c	2011-07-15 21:23:16.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     254,
  /**/

-- 
Eight Megabytes And Continually Swapping.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
